### ISSUE TITLE
Imbued Heart Not Triggering Loot Notifier

### ISSUE THREAD

**Issue by jdanthdavis** [pajlads/DinkPlugin#576]
### Checklist

- [X] I'm reporting a problem with Dink
- [X] I've searched the [issues and pull requests](https://github.com/pajlads/Dinkplugin/issues?q=) for similar looking reports

### Describe your issue

I received a Imbued Heart from a [Dreadborn Araxyte](https://oldschool.runescape.wiki/w/Dreadborn_Araxyte) and Dink did not fire a loot notification. However, I did receive a collection log notification.

Some more information: We do have history of receiving loot notifications for Imbued Hearts & Eternal Gems but these notifications were **not** from Dreadborn Araxytes. Because of this, I would assume an Eternal Gem would not fire if dropped from a Dreadborn Araxyte either.

### Screenshots

**In-game message**
![image](https://github.com/user-attachments/assets/072f3e0d-4fbf-43ca-9190-3fc6c09511b3)

**Dink settings - note: this notification sends directly to Discord**
![image](https://github.com/user-attachments/assets/c9f249cf-1a40-487c-8740-cbc1b88f3ca8)

### Runelite Logs

[client.log](https://github.com/user-attachments/files/17535051/client.log)

### Runelite Version

1.10.42

**Comment by iProdigy** [comment]
Does the loot show up in runelite's loot tracker ui?

**Comment by jdanthdavis** [comment]
> Does the loot show up in runelite's loot tracker ui?

It does appear to be missing. Runelite problem then?
![image](https://github.com/user-attachments/assets/5d496a54-04d6-400f-bd06-6420244adfbd)

**Comment by iProdigy** [comment]
> Runelite problem then? 

Yeah, I don't see an existing issue for this so you can report it at https://github.com/runelite/runelite/issues

**Comment by Felanbird** [comment]
I spoke to Cook (the wiki dev who gets the Loot Tracker data) and he confirmed he's gotten no tertiary drops in 50,000 kills, so it's definitely broken.
